tools/dptools/README.rst

```diff
@@ -60,10 +61,12 @@ individual scripts can be tested, for example by ::
 Installation
 ============
 
-Please note, that the package needs at least **Python 2.6** or later,
-with Python 3.X preferred. It additionally needs Numerical Python (the
-numpy module).
+Please note, that this package has been tested for **Python 3.X**
```

Comment:
you have a 3.10 requirement in the pyproject.toml, so is there an expected earliest 3.x that will work?

Reply:
The 3.10 requirement comes from our general support strategy, but I would expect the dptools to work for arbitrary Python 3.x.

Reply:
Shall we be more specific (e.g. Python >=3.10) in the README or keep the current version?